ksqldb-engine-common/src/test/java/io/confluent/ksql/schema/ksql/DefaultSqlValueCoercerTest.java

```diff
@@ -1081,6 +1081,7 @@ static SqlType typeInstanceFor(final SqlBaseType baseType) {
         .put(SqlBaseType.TIMESTAMP, new Timestamp(1535792475000L))
         .build();
 
+    @SuppressWarnings("fallthrough")
```

Comment:
Don't have a strong opinion -- it would be possible to either duplicate the code (worst option IMHO) or extract it into a method and call it for both cases. But using fall-through or calling a shared method (including parameter passing) does not make big difference to me with regard to simpler/more elegant code. If you feel strong about it, follow up PRs are welcome :)